search-app/src/cmr/search/services/provider_service.clj

```diff
@@ -0,0 +1,19 @@
+(ns cmr.search.services.provider-service
+  "Functions for searching operations on providers. All functions return
+  the underlying Metadata DB API clj-http response which can be used
+  as a Ring response."
+  (:require
+   [cmr.common.util :as u]
```

Comment:
I was once told to not do one letter names anymore

Reply:
Okay I'll change that one and I might as well change the one I copied it from